src/view/com/auth/LoggedOut.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
-import {SafeAreaView} from 'react-native'
+import {Pressable, SafeAreaView, StyleSheet} from 'react-native'
+import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome'
```

Comment:
Just moved the close button into this file so that it's present on all logged out screens